concepticondata/concepticon.tsv

```diff
@@ -3034,3 +3034,31 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3038	SAME-SEX YOUNGER SIBLING	Kinship	A person of the same sex who is born after another person who has the same parents.	Person/Thing	
 3039	DIFFERENT-SEX OLDER SIBLING	Kinship	A person of the different sex who is born before another person who has the same parents.	Person/Thing	
 3040	DIFFERENT-SEX YOUNGER SIBLING	Kinship	A person of the different sex who is born after another person who has the same parents.	Person/Thing	
+3041	PATERNAL AUNT (FATHER'S OLDER SISTER)	Kinship	The older sister of one's father.	Person/Thing	
+3042	MATERNAL AUNT (MOTHER'S OLDER SISTER)	Kinship	The older sister of one's mother.	Person/Thing	
```

Comment:
sorry to bother, but we'd need concept relations here:

3042 MATERNAL AUNT (MOTHER'S OLDER SISTER) broader 2693 MATERNAL AUNT (MOTHER'S SISTER)